Xamarin.Forms.Platform.UAP/BoxViewRenderer.cs

```diff
@@ -33,5 +46,11 @@ protected override AutomationPeer OnCreateAutomationPeer()
 
 			return new FrameworkElementAutomationPeer(Control);
 		}
+
+		private void SetCornerRadius(Core.CornerRadius cornerRadius)
+		{
+			Control.RadiusX = cornerRadius.TopLeft;
```

Comment:
Seems like this would ignore the values for the TopRight and BottomLeft corners, wouldn't it?

Reply:
Changed!